terraform/pilot/main.tf

```diff
@@ -15,15 +15,20 @@ variable "branch" {}
 
 variable "security_context" { default = "dev" }
 
-variable "aws_region" { default = "us-east-1" }
-variable "ami_type" { default = "hardened" }
+variable "network" { 
+    type = "map"
+    default = {
+        region = "us-east-1" 
+    }
+}
+variable "ami_type" { default = "not-hardened" }
 variable "bastion_subnet_id" {}
 variable "bastion_security_group_id" {}
-variable "bastion_instance_type" {}
+variable "bastion_instance_type" { default = "m3.medium" }
```

Comment:
According to [this](https://docs.google.com/spreadsheets/d/1YLiMhKgIMdqvTxlvjwA5MKfqlY-53oOyXm1VMrOoruE/edit#gid=200356483), looks like for our smallest instances we are using t2.small for dev, t2.medium for prod.